src/parquet/file/reader.h

```diff
@@ -100,11 +101,11 @@ class ParquetFileReader {
 
   // API Convenience to open a serialized Parquet file on disk
   static std::unique_ptr<ParquetFileReader> OpenFile(const std::string& path,
-      bool memory_map = true, MemoryAllocator* allocator = default_allocator());
+      ReaderProperties opts, bool memory_map = true);
```

Comment:
Given that we often just want to use the sane defaults, wouldn't it be better user experience if we could provide here a default instantiation?  

Then we would not need to clutter the code with `ReaderProperties opts;` lines just to get the defaults.